Rubberduck.Deployment.Build/RubberduckPostBuildTask.cs

```diff
@@ -0,0 +1,365 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Tasks;
+using Microsoft.Build.Utilities;
+using Microsoft.VisualStudio.Setup.Configuration;
+using Rubberduck.Deployment.Build.Builders;
+using Rubberduck.Deployment.Build.IdlGeneration;
+using Rubberduck.Deployment.Build.Structs;
+using Rubberduck.Deployment.Build.Writers;
+
+namespace Rubberduck.Deployment.Build
+{
+    internal struct DllFileParameters
+    {
+        public string DllFile { get; }
+        public string IdlFile { get; }
+        public string Tlb32File { get; }
+        public string Tlb64File { get; }
+        public string SourceDll { get; }
+        public string TargetDll { get; }
+        public string SourceTlb32 { get; }
+        public string TargetTlb32 { get; }
+        public string SourceTlb64 { get; }
+        public string TargetTlb64 { get; }
+        public string DllXml { get; }
+        public string TlbXml { get; }
```

Comment:
consider explicitly marking these as `readonly`

Reply:
Set the `struct` to `readonly`.

Reply:
@Vogel612 they're immutable auto-properties. This would be identical AFAIK:

```csharp
private readonly _tlbXml;
public string TlbXml => _tlbXml;
```